web/angular-wallet/src/app/layout/components/burst-fee-selector/burst-fee-selector.component.ts

```diff
@@ -12,18 +12,23 @@ import { Options } from 'ng5-slider';
 export class BurstFeeSelectorComponent implements OnInit {
   @Input('fees') fees: SuggestedFees;
 
-  feeNQTValue = '';
+  feeNQTValue = 0;
   options: Options;
 
   @Input()
-  get feeNQT(): string {
+  get feeNQT(): number {
     return this.feeNQTValue;
   }
 
   @Output()
   feeNQTChange = new EventEmitter();
-  set feeNQT(feeNQT: string) {
-    this.feeNQTValue = feeNQT;
+
+  set feeNQT(feeNQT: number) {
+    if (!feeNQT) {
+      this.feeNQTValue = 0;
+      return;
+    }
+    this.feeNQTValue = parseFloat(feeNQT.toString());
```

Comment:
had to do some stuff here to ensure the data is sanitized while pleasing typescript

Reply:
ok, I remember I had similar things to do for the send-warn dialog